Adyen/Core/Components/Base/Component.swift

```diff
@@ -78,9 +78,9 @@ public extension Component {
 }
 
 private enum AssociatedKeys {
-    internal static var isDropIn = "isDropInObject"
+    internal static var isDropIn: Void?
```

Comment:
This solution's readability could be great. Shall we either add a comment as in the link you mentioned?
```swift
private enum AssociatedKeys {
    internal static var isDropIn: Void? // "isDropin"
}
```